modules/cli/src/main/scala/scala/cli/commands/config/ConfigOptions.scala

```diff
@@ -27,10 +27,18 @@ final case class ConfigOptions(
   @Tag(tags.inShortHelp)
     dump: Boolean = false,
   @Group(HelpGroup.Config.toString)
-  @HelpMessage("Create PGP key in config")
+  @HelpMessage("Create PGP keychain in config")
+  @Tag(tags.inShortHelp)
   @Tag(tags.experimental)
     createPgpKey: Boolean = false,
   @Group(HelpGroup.Config.toString)
+  @HelpMessage("A password used to encode the private PGP keychain")
+  @Tag(tags.inShortHelp)
+  @Tag(tags.restricted)
```

Comment:
everything tied to `publish` should be considered experimental for now, I think
```suggestion
  @Tag(tags.experimental)
```